feat(HTTP): Refresh request method pages (#35745)
mdn/content: files/en-us/web/http/methods/get/index.md

{{HTTPSidebar}}

> The **HTTP `GET` request method** requests a representation of the specified resource.

Reviewer: ```suggestion
The **`GET`** HTTP method requests a representation of the specified resource.
```

Requests using `GET` should only be used to request data and shouldn't contain a body.

> [!NOTE]
> The semantics of sending a message body in `GET` requests are undefined.

> > Some systems may reject the request with a [4XX client error](/en-US/docs/Web/HTTP/Status#client_error_responses) response.

Reviewer: ```suggestion
> Some servers may reject the request with a [4XX client error](/en-US/docs/Web/HTTP/Status#client_error_responses) response.
```

> GET /index.html

Reviewer: Again, maybe we should be generic and refer to a token here. Or maybe not. Up to you.

```

## Examples

### Successfully retrieving a resource

> The following `GET` request asks for `example.com/contact`:

Reviewer: ```suggestion
The following `GET` request asks for the resource at `example.com/contact`:
```

```http
GET /contact HTTP/1.1
Host: example.com
User-Agent: curl/8.6.0
Accept: */*
```

> The server sends back the resource with a {{HTTPStatus("200", "200 OK")}} response:

Reviewer: with an Xxxx response is probably fine, but to be more precise ...

```suggestion
The server sends back the resource with a {{HTTPStatus("200", "200 OK")}} status code, indicating success:
```
